assets/javascripts/spoiler.js

```diff
@@ -16,103 +8,36 @@
-    // spoiler text is gray so as to maintain plugin compatibility with both light and dark theme sites.
-    var textShadow = "gray 0 0 " + radius + "px",
-      userSelect = "-webkit-user-select";
-
-    if (isIE) {
-      textShadow =
-        radius <= 0 ? "0 0 0 0 gray" : "0 0 " + radius + "px .1px gray";
-      userSelect = "-ms-user-select";
-    }
-    if (isFirefox) {
-      userSelect = "-moz-user-select";
-    }
-
-    $spoiler
-      .css("background-color", "transparent")
-      .css("color", "rgba(0,0,0,0)")
-      .css("text-shadow", textShadow)
-      .css(userSelect, "none");
-  }
-
-  function blurSelector($sel, radius) {
-    var value = radius > 0 ? "blur(" + radius + "px)" : "";
-    if (isIE) {
-      $sel.css(
-        "-ms-filter",
-        "progid:DXImageTransform.Microsoft.Blur(pixelradius=" + radius + ")"
-      );
-    } else {
-      $sel.css("filter", value).css("-webkit-filter", value);
-    }
-  }
-
-  function blurLinkAndPre($spoiler, radius) {
+  function blurText($spoiler) {
+    $spoiler.addClass("spoiler-blurred");
     $("a", $spoiler).addClass("no-track-link");
-
-    $("a,pre", $spoiler).each(function(index, linkOrPre) {
-      blurSelector($(linkOrPre), radius);
-    });
-  }
-
-  function blurImageIE($spoiler, radius) {
-    var $images = $("img", $spoiler);
-    if (isIE) {
-      $images.css("opacity", radius === 0 ? "1.0" : "0.0");
-    } else {
-      blurSelector($images, radius);
-    }
   }
 
-  var applyBlur = function($spoiler, option) {
+  var applyBlur = function($spoiler) {
     blurLazyYT($spoiler);
-    blurText($spoiler, option.text);
-    blurLinkAndPre($spoiler, option.link);
-    blurImageIE($spoiler, option.image);
+    blurText($spoiler);
   };
 
-  var applySpoilers = function($spoiler, options) {
-    var maxBlur = options.max,
-      partialBlur = options.partial,
-      noBlur = options.none;
-
+  var applySpoilers = function($spoiler) {
     $spoiler.data("spoiler-state", "blurred");
 
-    applyBlur($spoiler, maxBlur);
-
-    $spoiler
-      .on("mouseover", function() {
-        $spoiler.css("cursor", "pointer");
-        if ($spoiler.data("spoiler-state") === "blurred") {
-          applyBlur($spoiler, partialBlur);
-        }
-      })
-      .on("mouseout", function() {
-        if ($spoiler.data("spoiler-state") === "blurred") {
-          applyBlur($spoiler, maxBlur);
-        }
-      })
-      .on("click", function(e) {
-        if ($spoiler.data("spoiler-state") === "blurred") {
-          $spoiler.data("spoiler-state", "revealed").css("cursor", "auto");
-          applyBlur($spoiler, noBlur);
-          e.preventDefault();
-        } else {
-          if (e.target && e.target.tagName !== "A") {
-            $spoiler.data("spoiler-state", "blurred").css("cursor", "pointer");
-            applyBlur($spoiler, partialBlur);
-            e.preventDefault();
-          }
-        }
-      });
+    applyBlur($spoiler);
+
+    $spoiler.on("click", function(e) {
```

Comment:
How are blurred links handled? The workflow was 

1. blurred
2. change blur to partial when mouseover
3. remove blue on first click
4. open link on 2nd click